index.md

```diff
@@ -19,7 +19,7 @@ customers happy.
     </ul>
     <div class="card-bottom">
       <div class="card-button">
-        <button class="hxBtn hxBtn--primary">Full Designer Guide</button>
+        <button class="hxBtn hxBtn--primary"><a href="{{site.baseurl}}/overview/design-getting-started.html">Full Designer Guide</a></button>
```

Comment:
update link

Reply:
I checked at if we simply apply the `hxBtn hxBtn--primary` classes to an `<a>` tag the font-color will not be correctly displayed. I removed the override to hxBtn and instead based the `<a>` selection based on the following selection path;  .card-button -> button -> a. This has the same behavior and scope, but does not violate the rules of the helix-ui

Reply:
Sounds good!